tests/integ/test_packaging.py

```diff
@@ -343,6 +391,34 @@ def import_scikit_fuzzy() -> str:
     Utils.check_answer(session.sql(f"select {udf_name}()"), [Row("0.4.2")])
 
 
+@pytest.mark.skipif(
+    IS_IN_STORED_PROC,
+    reason="Subprocess calls are not allowed within stored procedures",
+)
+def test_add_requirements_unsupported_usable_by_sproc(session, resources_path):
```

Comment:
This test is moved into `test_packaging.py` from `test_stored_procedure.py` file as it references the `scikit-fuzzy` package